package.json

```diff
@@ -14,25 +14,25 @@
   "dependencies": {
     "@expo-google-fonts/dev": "^0.1.0",
     "@expo-google-fonts/open-sans": "^0.1.0",
-    "@react-native-community/masked-view": "0.1.10",
+    "@react-native-masked-view/masked-view": "0.2.6",
```

Comment:
We needed to upgrade for react 17 support, and they changed their name with [v0.2.0](https://github.com/react-native-masked-view/masked-view/releases/tag/v0.2.0) because of https://github.com/react-native-community/discussions-and-proposals/issues/176.